assets/section-blog-post.css

```
@@ -2,14 +2,159 @@
   margin-top: 5rem;
 }
 
+.article-details--wrap {
+  display: inline-grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  row-gap: 15px;
+  column-gap: 10px;
+  padding-top: 10px;
+}
+
+.article-breadcrumbs {
+  display: flex;
+  gap: 8px;
+  align-items: center;
+  padding-bottom: 10px;
+}
+
+.article-breadcrumbs--link {
+  display: block;
+  font-size: 1.2rem;
+  font-weight: 700;
+  text-underline-offset: 3px;
+  text-decoration: underline;
+  text-transform: uppercase;
+  position: relative;
+  color: rgb(var(--color-foreground));
+}
+
+.article-breadcrumbs--arrow {
+  width: 11px;
+  height: auto;
+}
+
+.article-details-item--author {
+  grid-area: 1 / 1 / 2 / 3;
+}
+
+.article-details-item--date {
+  grid-area: 2 / 1 / 3 / 2;
+}
+
+.article-details-item--estimated-time {
+  grid-area: 2 / 2 / 3 / 3;
+}
+
+.article-details--wrap > span {
+  text-transform: capitalize;
+  font-size: 1.6rem;
+  line-height: normal;
+}
+
+.article-details-item--estimated-time.line-divider {
+  position: relative;
+}
+
+.article-details-item--estimated-time.line-divider:after {
+  content: '';
+  height: 16px;
+  width: 1px;
+  background: rgb(234, 234, 234);
+  position: absolute;
+  left: -6px;
+  top: 50%;
+  will-change: transform;
+  transform: translateY(-50%) translateZ(0);
+}
+
+.article--socials {
+  padding: 20px 15px;
+  border-top: 1px solid rgb(245, 245, 245);
+  border-bottom: 1px solid rgb(245, 245, 245);
+}
+
+.article--socials-heading {
+  font-size: 1.6rem;
+  color: rgb(var(--color-foreground));
+  margin: 0;
+}
+
+.article--socials-icons {
+  display: flex;
+  gap: 6px;
+  padding-top: 10px;
+}
+
+.article--socials-icons--center {
```

Good place to use a modifier from BEM

```suggestion
.article-socials__icons--center {
```